DOc/docs/source/Options/user-story-4.rst

```diff
@@ -10,22 +10,43 @@ Find data fast and observed quality metrics
-(injected humour and the like).
 
+Next to data governance, data quality is another core pillar of data management. 
+Good data quality helps make data more useful and usable, resulting in better business outcomes.
+Meanwhile, bad data quality limits the value of data and results in worse business outcomes. 
 
+Data quality can be measured in many ways. 
+It is the responsibility of the data governance organisation to define data quality metrics.
+Data quality is typically measured along the following dimensions:
+
+| Dimension    | Description                                                                                                             |
+|--------------|-------------------------------------------------------------------------------------------------------------------------|
+| Accuracy     | Whether or not the data contains errors. For example, a person's name should always be correct.                         |
+| Completeness | Whether or not each row of the data has a value. For example, a person should always have a name.                       |
+| Timeliness   | Whether or not the data is up to date. For example, the data should only include people currently employed.             |
+| Uniqueness   | Whether or not values that need to be unique, actually are unique. For example, every employee should have a unique ID. |
+| Validity     | Whether or not the data fits with a predefined format. For example, a person's initials should always be capitalized.   |
+
+For every dimension, one or more data quality rules can be defined.
+For every rule, a percentage score is assigned. 
+A score of 100% indicates that the dataset is perfectly compliant with that particular rule.
+
+A lower data quality score for a particular rule indicates that there are data quality issues.
+The impact and consequences of these issues depends on the characteristics of the dataset.
+
+Typically, manually entered data tends to be of lesser quality than automatically generated data.
+Improving the data quality of manually entered data, tends to involve the education of people and process improvements.
+Meanwhile, lower data quality scores for automatically generated data may indicate a bug in the system.
+
+Data quality should be measured at every system where the data is stored, since data transformations can cause data quality to change.
+If there are data quality issues, these should be solved at the point where the data quality issues are introduced.
+This is typically at the place where the master data is stored.
+
+Aurelius Atlas allows for the discovery and insight of an organisation's data quality across data sources.
+This allows for better business outcomes for users of data.
+The tool also allows the people responsible for data quality, to do root cause analysis of data quality issues.
+
+The following example demonstrates how data of good quality is discovered by a data scientist.
 
```

Comment:
This is a detail/lengthy introduction of what data governance is and how we look at it. The first two sentences state what is the problem, the subsequent sentence states why it is a problem. The rest addresses how we solve the problem.
Can you add some more details on why it is a problem. What are the consequences if data quality is bad?
AT the end I am missing a description of a short scenario: Something like a data engineer has found the dataset NL1-HR-001 <add link> and wants to understand the data quality of the dataset.